### ISSUE TITLE
token间没有space的情况

### ISSUE THREAD

**Issue by chuyi369** [01-ai/Yi-1.5#38]
请问我在使用Yi-1.5-9B-Chat进行推理时，出现token间没有space的情况，这是bug吗？

transformers=4.47

                outputs = self.model(input_ids=input_ids, attention_mask=attention_mask, use_cache=True)
                next_token_logits = outputs.logits[:, -1, :] / temp
                next_token_logits = next_token_logits.to(self.model.device) 
                next_token_logits = self.filter_forbidden_tokens(next_token_logits)
                next_token_logits = logits_processor(input_ids, next_token_logits)
                output_tokens += 1
                step += 1

                next_token_id = torch.multinomial(F.softmax(next_token_logits, dim=-1), num_samples=1)

                next_word = self.tokenizer.decode(next_token_id[0], skip_special_tokens=True)

                response += next_word

<img width="1016" alt="截屏2024-12-25 17 49 21" src="https://github.com/user-attachments/assets/e9d3021c-c79d-4cc7-b418-f01391d18b20" />

**Comment by chuyi369** [comment]
谢谢我解决了这个问题，通过修改tokenizer_config.json中add_prefix_space为false

参考：https://github.com/01-ai/Yi-1.5/issues/36
